app/views/_includes/forms/training-details.html

```diff
@@ -6,7 +6,7 @@ <h1 class="govuk-heading-l">{{pageHeading}}</h1>
 
 {% set commencementDateInputHtml %}
   {% set hintHtml %}
-    When this trainee started, rather than the programme start date.<br>
+    This may be different to your overall programme start date. Give the date that the trainee evidenced that they are on the course.<br>
```

Comment:
I feel like providers might struggle with 'evidenced'. I think saying it's when the trainee started is clearer.

```suggestion
    When the trainee started on the course. This may be different to your overall programme start date.<br>
```

Reply:
I still don't know what we mean by 'started' - this seems really unclear to me.

Reply:
It's the date the trainee started on the course. The course / programme start date is the 'official' start date for the course, but might only be as accurate to the nearest month. The trainee commencement date is when they actually started.

Reply:
I agree 'evidenced' is not the right term, it sounds made up! 

What about something where we reference the individual?

> This may be different to your overall programme start date. Give the date that the trainee **began their** training.